go.mod

```diff
@@ -271,8 +271,10 @@ replace (
 	github.com/evmos/evmos/v16 => github.com/ExocoreNetwork/evmos/v16 v16.0.3-0.20240828081344-d5cfcd34a812
```

Comment:
Are the changes in this file (ledgerwatch/interfaces and websocket) a result of a security advisory?

Reply:
fixed